test/broadcastable/test_helper_test.rb

```diff
@@ -1,263 +1,269 @@
-    message = Message.new(id: 1)
-
-    message.broadcast_replace
-    message.broadcast_remove
-    replace, remove, *rest = capture_turbo_stream_broadcasts message
-
-    assert_empty rest
-    assert_equal "replace", replace["action"]
-    assert_equal "remove", remove["action"]
-    assert_not_empty replace.at("template").element_children
-    assert_nil remove.at("template")
-  end
-
-  test "#capture_turbo_stream_broadcasts returns <turbo-stream> elements broadcast on an Array of stream objects" do
-    message = Message.new(id: 1)
-
-    message.broadcast_replace_to [message, :special]
-    message.broadcast_remove_to [message, :special]
-    replace, remove, *rest = capture_turbo_stream_broadcasts [message, :special]
-
-    assert_empty rest
-    assert_equal "replace", replace["action"]
-    assert_equal "remove", remove["action"]
-    assert_not_empty replace.at("template").element_children
-    assert_nil remove.at("template")
-  end
-
-  test "#capture_turbo_stream_broadcasts returns <turbo-stream> elements broadcast on a stream name from a block" do
-    message = Message.new(id: 1)
-
-    replace, remove, *rest = capture_turbo_stream_broadcasts "messages" do
-      message.broadcast_replace_to "messages"
-      message.broadcast_remove_to "messages"
-    end
-
-    assert_equal "replace", replace["action"]
-    assert_equal "remove", remove["action"]
-    assert_empty rest
-  end
-
-  test "#capture_turbo_stream_broadcasts returns <turbo-stream> elements broadcast on a stream object from a block" do
-    message = Message.new(id: 1)
-
-    replace, remove, *rest = capture_turbo_stream_broadcasts message do
-      message.broadcast_replace
-      message.broadcast_remove
-    end
-
-    assert_empty rest
-    assert_equal "replace", replace["action"]
-    assert_equal "remove", remove["action"]
-    assert_not_empty replace.at("template").element_children
-    assert_nil remove.at("template")
-  end
-
-  test "#capture_turbo_stream_broadcasts returns <turbo-stream> elements broadcast on an Array of stream objects from a block" do
-    message = Message.new(id: 1)
-
-    replace, remove, *rest = capture_turbo_stream_broadcasts [message, :special] do
-      message.broadcast_replace_to [message, :special]
-      message.broadcast_remove_to [message, :special]
-    end
-
-    assert_empty rest
-    assert_equal "replace", replace["action"]
-    assert_equal "remove", remove["action"]
-    assert_not_empty replace.at("template").element_children
-    assert_nil remove.at("template")
-  end
-
-  test "#capture_turbo_stream_broadcasts returns an empty Array when no broadcasts happened on a stream name from a block" do
-    streams = capture_turbo_stream_broadcasts "messages" do
-      # no-op
-    end
-
-    assert_empty streams
-  end
-end
+module Turbo
+  module Broadcastable
+    module TestHelper
+      class CaptureTurboStreamBroadcastsTest < ActiveSupport::TestCase
```

Comment:
This is necessary because `Turbo::Broadcastable::TestHelper` isn't loaded and therefore the namespace isn't available until after the `include Turbo::Broadcastable::TestHelper`, due to lazy loading. We could use `if defined?(ActiveJob)`, rather than `ActiveSupport.on_load(:active_job)`, but that doesn't seem better.